### PR TITLE
feat: add calculator user interface

### PR DESCRIPTION
**Pull request by Amrhub**
- [x]  Deleted all the boilerplate from CRA (text, images, styles).
- [x] Created a directory called components.
- [x] Inside components, created a new `Calculator.js` `CalculatorKeypadRow.js` file. 
- [x] `Calculator.js` renders the calculator user interface matching provided design, where, `CalculatorKeypadRow.js` just for rendering keypads to easy readability of code
- [x]  Used class-based components.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App flex-center-center">
+      <Calculator />
```
Comment (omar-labana) [inline src/App.js]: - [x] The App.js component should be class-based.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Amrhub, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Amrhub) [reply src/App.js]: I am sorry I don't understand your review, could you explain, please?

**Review by er-contreras — APPROVED** [review]
Hi @Amrhub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**